lib/engines/content_object_store/test/unit/app/services/get_host_content_items_test.rb

```diff
@@ -60,31 +66,23 @@ class ContentObjectStore::GetHostContentItemsTest < ActiveSupport::TestCase
       assert_equal result[0].publishing_organisation, expected_publishing_organisation
     end
 
-    it "returns an empty array if the request fails" do
-      document = mock("content_block_document", content_id: target_content_id)
-
-      stub_request(:get, expected_url)
-        .to_return(body: "", status: 500)
-
-      result = described_class.by_embedded_document(content_block_document: document)
-
-      assert_equal result, []
-    end
-
-    it "depends on Plek to set the right hostname" do
-      document = mock("content_block_document", content_id: target_content_id)
-
-      plek_domain = "https://foo.dev.gov.uk"
-      Plek.stubs(:find).with("publishing-api").returns(plek_domain)
-
-      stub_request(:get, %r{\A#{plek_domain}/v2/content/#{target_content_id}/embedded})
-        .to_return(body: response_body.to_json, status: 200)
-
-      described_class.by_embedded_document(
-        content_block_document: document,
+    it "returns an error if the content that embeds the target can't be loaded" do
```

Comment:
so if this request fails, are we not able to see the relevant embedded Document at all, whereas before we would have failed silently and shown an empty list?

Reply:
Yup, if it fails now the host document page won't load at all.

The previous behaviour was me skipping the problem of error handling as I knew we'd be coming back to take out the net http request. 

I think this fail safe feels a reasonable starting place. In future we could change things so we still render the page but show an error message where the table was.